lib/reverse_sentence.rb

```diff
@@ -1,4 +1,55 @@
 # A method to reverse the words in a sentence, in place.
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  return my_sentence if my_sentence.nil?
+
+  i = 0
```

Comment:
Line 5 through 14 are really reversing the string. To avoid duplicating code, you could simply call your reverse_word method here.